cypress/e2e/calendar/add-calendar-never-close.cy.js

```diff
@@ -0,0 +1,144 @@
+import { Select as BaseSelect } from '@interactors/html';
+import {
+  TextField, HTML, Calendar, Accordion, MultiSelect, Pane, Select,
+  MultiColumnListCell, MultiColumnListRow, Button, including, Link, IconButton, matching, TimeDropdown
+} from '../../../interactors';
+
+import calendarFixtures from '../../fixtures/calendar_e2e_fixtures';
+
+const testServicePoint = calendarFixtures.servicePoint;
+const testCalendar = calendarFixtures.calendar;
+const newCalendarInfo = calendarFixtures.data.newCalendar;
+
+
+const toggleCalendarButton = HTML.extend('toggle calendar button').selector('button[id^=datepicker-toggle-calendar-button-dp-]');
+
+let calendarID;
+
+describe('Create calendars that are 24/7 (never close)', () => {
+  before(() => {
+    // login
+    cy.loginAsAdmin();
+
+    // reset db state
+    cy.deleteServicePoint(testServicePoint.id, false);
+
+    // create test service point
+    cy.createServicePoint(testServicePoint, (response) => {
+      testCalendar.assignments = [response.body.id];
+    });
+
+    // cy.deleteCalendarUI();
+    cy.openCalendarSettings();
+  });
+
+  after(() => {
+    cy.deleteServicePoint(testServicePoint.id, true);
+    if (calendarID) {
+      cy.deleteCalendar(calendarID);
+    }
+  });
+
+
+  it('adds new calendar', () => {
+    cy.do([
```

Comment:
don't use cy.do inside test. Use it inside fragments please.
Create a common funtion in fragments, import it in test and use it